sentry_sdk/integrations/starlite.py

```diff
@@ -176,66 +176,67 @@ def patch_http_route_handle() -> None:
     async def handle_wrapper(
         self: "HTTPRoute", scope: "HTTPScope", receive: "Receive", send: "Send"
     ) -> None:
-        hub = Hub.current
-        integration: StarliteIntegration = hub.get_integration(StarliteIntegration)
+        integration: StarliteIntegration = sentry_sdk.get_client().get_integration(
+            StarliteIntegration
+        )
         if integration is None:
             return await old_handle(self, scope, receive, send)
 
-        with hub.configure_scope() as sentry_scope:
-            request: "Request[Any, Any]" = scope["app"].request_class(
-                scope=scope, receive=receive, send=send
+        sentry_scope = SentryScope.get_isolation_scope()
+        request: "Request[Any, Any]" = scope["app"].request_class(
+            scope=scope, receive=receive, send=send
+        )
+        extracted_request_data = ConnectionDataExtractor(
+            parse_body=True, parse_query=True
+        )(request)
+        body = extracted_request_data.pop("body")
+
+        request_data = await body
+
+        def event_processor(event: "Event", _: "Dict[str, Any]") -> "Event":
```

Comment:
Instead of `"Dict[str, Any]"` we can use the `sentry_sdk.types.Hint` type

Reply:
Nice!